c2corg_ui/templates/helpers/view.html

```diff
@@ -6,6 +6,61 @@
     from c2corg_ui.views.document import get_slug
 %>
 
+<%def name="get_licence(doc)">\
+<%
+  doctype = doc['type']
+  CC_ND_NC = False
+  CC_BY_SA = False
```

Comment:
Instead of ``CC_ND_NC`` (by the way "BY" is missing here) and ``CC_BY_SA`` you should rather figure out if the document is "personal" or "collaborative" (or "copyright" for images).

And then in the HTML code you decide what icons to use depending on this.